src/python/grpcio/grpc/_cython/_cygrpc/credentials.pxd.pxi

```diff
@@ -57,6 +57,16 @@ cdef class ChannelCredentials:
   cdef grpc_channel_credentials *c_credentials
 
 
+cdef class SSLSessionCache:
+
+  cdef grpc_ssl_session_cache *_cache
+
+
+cdef class SSLSessionCacheLRU(SSLSessionCache):
```

Comment:
Drop this? An empty class is always a code smell, and there should probably be only one Cython-layer class per Core object type anyways.